readme/usage.md

```diff
@@ -99,6 +109,13 @@ while the joystick turns the view (depending on the game).
 Tested with the XBOX 360 Gamepad. On Ubuntu, gamepads worked better in
 Wayland than with X11 for me.
 
+## Foot Pedals
+
+Foot pedal clicks can be translated into mouse scroll events even if the
+device didn't support that. Tested on Kinesis Savant Elite 2 (3-pedal
+version) on Wayland/Ubuntu. Scroll events generated while holding the
+pedal linearly auto-repeat as they should.
```

Comment:
But that applies to all kinds of events (multimedia, keys, numbers, etc.), not just scrolling and not just foot-pedals. This is basically the whole purpose of key-mapper, so it should be written somewhere else. Maybe a generalized sentence can be placed at the top of usage.md, like "In theory any click, joystick or wheel event of any device can be mapped to any of the available key names, even if the device didn't support that", if you feel like it wasn't clear already.

> Scroll events generated while holding the pedal linearly auto-repeat as they should.

Basically https://github.com/sezanzeb/key-mapper/blob/main/readme/macros.md already says "keeps scrolling down while held"

Reply:
Aha, so the idea behind the current documentation is to keep it brief and normalized/deduplicated, do I understand it correctly? My initial impression from the docs was they are too terse and written from the developer point of view which doesn't that much care about highlighting practical usage of such great tool for someone who just discovered it and is trying to judge if the tool fits into his needs -- or is not aware of the full potential of such tool.

Would it make sense to you to split the docs into technical description of commandline options, syntax of key names and macros etc. (existing docs) -- and into (newly created) howto guide which would showcase concrete examples of use with different types of devices? Perhaps it could also improve the discoverability of this tool via web search engines.

Reply:
ok, lets add another doc file for it.

As far as usage.md and macros.md goes yeah, I think I tried to have that a bit generalized. There is a little bit of example stuff in a few places, but nothing that gives very concrete examples like "on this trackball model if you map that you can have this and that and it even works as foo".